project_template/spec/spec_helper.coffee

```diff
@@ -1,3 +1,29 @@
 Backbone = require('backbone')
 # Backbone.$ is not automatically set when backbone is loaded as CommonJS module
 $ = Backbone.$ = require('jquery')
+
+# Allow to use memo-isation in tests
+#
+# describe 'some feature', ->
+#
+#   data = memo().is -> { foo: 'bar' }
+#
+#   it 'shows bar', ->
+#     expect(data().foo).to.eql 'bar'
+#
+#   describe 'when undefined', ->
+#     data.is -> { foo: undefined }
+#
+#     it 'shows undefined', ->
+#       expect(data().foo).to.be.undefined
+#
+window.memo = require('memo-is')
+
+When = require('when')
+window.wait = (delay = 1) ->
```

Comment:
Let's talk (offline?) about in which situations you encounter this. I've seen this but often times the better/simpler approach is removing the asynchronicity from the test.

I do agree that there will always be cases where this is needed, I just think that we should encourage to use different approaches whenever possible.